test/md040-allowlist.md

```diff
@@ -0,0 +1,29 @@
+# fenced_code_blocks
```

Comment:
Love the tests! My new convention is to make the top level heading correspond to the file name.